example/Vagrantfile

```diff
@@ -5,100 +5,134 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-
-    # RackHD SERVER
+    
+    # MONORAIL SERVER
```

Comment:
any reasons modify RackHD to MONORAIL, I'm afraid users will be confused when they saw monorail